src/daemons/GraphDaemon.cpp

```diff
@@ -141,21 +143,49 @@ int main(int argc, char *argv[]) {
     try {
         gServer->serve();  // Blocking wait until shut down via gServer->stop()
     } catch (const std::exception &e) {
+        nebula::WebService::stop();
         FLOG_ERROR("Exception thrown while starting the RPC server: %s", e.what());
         return EXIT_FAILURE;
     }
 
+    nebula::WebService::stop();
     FLOG_INFO("nebula-graphd on %s:%d has been stopped", localIP.c_str(), FLAGS_port);
 
     return EXIT_SUCCESS;
 }
 
 
+Status installHandler(int signum, sighandler_t handler) {
```

Comment:
Please check a signal handler in src/common/SignalHandler

Reply:
> Please check a signal handler in src/common/SignalHandler

I am sorry that I can not find file named SignalHandler in src/common, Do you mean add a new file named SignalHandler in src/common to erase repeat code of “installHandler”?

Reply:
src/common/base

Reply:
> src/common/base

good point, thx, fixed